code/modules/holodeck/computer.dm

```diff
@@ -242,12 +245,12 @@ and clear when youre done! if you dont i will use :newspaper2: on you
 	for (var/_atom in spawned)
 		var/atom/atoms = _atom
 
-		if (isturf(atoms) || istype(atoms, /obj/effect/overlay/vis)) //ssatoms
+		if (isturf(atoms) || istype(atoms, /obj/effect/overlay/vis) || !atoms)
```

Comment:
Change the loop to `for (var/atom/atoms in spawned)` so that it filters out the nulls.

Reply:
done